src/netbase.cpp

```diff
@@ -68,6 +68,8 @@ bool static LookupIntern(const char *pszName, std::vector<CNetAddr>& vIP, unsign
 
     {
         CNetAddr addr;
+        // We can't actually resolve onion addresses, return a network address
```

Comment:
^ above, nit: maybe ParseNetwork could get also a little comment on which networks are currently supported due to addr v1, notably diff between tor and onion

Reply:
Addressed in https://github.com/bitcoin/bitcoin/pull/16029